regions_www/tabs/download_national.Rmd

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
Having `Rmd` files in a directory that was served by shiny server caused strange problems before, just one to watch if needs be.

Reply:
Seems to work fine locally. Yes one to watch.

Reply:
Locally it will not be an issue as locally we don't run shiny server, we run shiny in R.

Reply:
right, thanks. aanother reason to switch to static generation of the download pages, if this cause any issues